hyper/hyper/hyper.go

```diff
@@ -10,25 +10,38 @@ import (
 
 // Interface is an interface with all the actions that a hypervisor has to be able to do
 type Interface interface {
-	// Get returns a desktop using it's name
+	// Get returns a running desktop object from it's name
 	Get(name string) (*libvirt.Domain, error)
 
-	// Start starts a new machine using the provided XML
+	// Start starts a new machine using the provided XML definition
+	// so it is a non-persistent desktop from libvirt point of view
 	Start(xml string, options *StartOptions) (*libvirt.Domain, error)
 
 	// Stop stops a running desktop
 	Stop(desktop *libvirt.Domain) error
 
-	// XMLGet returns the XML definition of a desktop
+	// Suspend suspends a running desktop temporarily saving its memory state. It won't persist hypervisor restarts.
```

Comment:
"It won't persist if the hypervisor restarts"

Reply:
?